djangocms_version_locking/check.py

```diff
@@ -1,18 +1,6 @@
-from django.contrib.contenttypes.models import ContentType
+from .helpers import content_is_unlocked
 
-from .models import VersionLock
 
-
-def content_is_unlocked(placeholder, user):
-    """Pass the check if lock doesn't exist or
-    object is locked to provided user.
-    """
+def placeholder_content_is_unlocked(placeholder, user):
```

Comment:
I guess this could be in the helper too, may be no need for the check.py file in that case. See what @chaosk thinks.

Reply:
Personally I _really_ dislike having modules called helpers/utils, which provide no context as to what they contain. I'd much rather have 5 distinctly named files instead. That being said feel free to move it to helpers, as that's the convention for the gaps apps.

Reply:
I will add this to the helpers.py :)